components/Common/CodeBox/index.stories.tsx

```diff
@@ -0,0 +1,130 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import { MDXRemote } from 'next-mdx-remote';
+import { serialize } from 'next-mdx-remote/serialize';
+import rehypePrettyCode from 'rehype-pretty-code';
+import type { HighlighterOptions } from 'shiki';
+import { getHighlighter, setCDN } from 'shiki';
+import shikiNordTheme from 'shiki/themes/nord.json';
+
+import CodeBox from '@/components/Common/CodeBox/index';
+import codeTabsResolver from '@/mdx/plugins/codeTabsResolver.mjs';
+import { mdxComponents } from '@/providers/mdxProvider';
+import { SUPPORTED_LANGUAGES } from '@/shiki.config.mjs';
+
+setCDN('https://unpkg.com/shiki');
+
+type Story = StoryObj<typeof CodeBox>;
+type Meta = MetaObj<typeof CodeBox>;
+
+export const SingleFile: Story = {
+  args: {
+    children: `\`\`\`javascript showLineNumbers
+const http = require('http');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World');
+});
+
+server.listen(port, hostname, () => {
+  console.log(\`Server running at http://\${hostname}:\${port}/\`);
+});
+\`\`\``,
+  },
+};
+
+export const MultipleFiles: Story = {
+  args: {
+    children: `\`\`\`mjs|cjs showLineNumbers labels="MJS,CJS"
+const { createHmac } = await import('node:crypto');
+
+const secret = 'abcdefg';
+const hash = createHmac('sha256', secret)
+               .update('I love cupcakes')
+               .digest('hex');
+console.log(hash);
+// Prints:
+//   c0fa1bc00531bd78ef38c628449c5102aeabd49b5dc3a2a516ea6ea959d6658e
+--------------
+const { createHmac } = require('node:crypto');
+
+const secret = 'abcdefg';
+const hash = createHmac('sha256', secret)
+               .update('I love cupcakes')
+               .digest('hex');
+console.log(hash);
+// Prints:
+//   c0fa1bc00531bd78ef38c628449c5102aeabd49b5dc3a2a516ea6ea959d6658e
+\`\`\``,
+  },
+};
+
+export const WithMoreOptions: Story = {
+  args: {
+    children: `\`\`\`bash|sh showLineNumbers labels="MacOS / Linux (nvm),Windows (Chocolatey)" link="[More options](https://github.com/nodejs/nodejs.org)"
+# Install Node Version Manager (NVM)
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/master/install.sh | bash
+
+# Install Node.js
+nvm install --lts
+
+# Check that Node is installed
+node -v
+
+# Check your NPM version
+npm -v
+--------------
+# Install Node Version Manager (NVM)
+choco install nvm
+
+# Install Node.js
+nvm install --lts
+
+# Check that Node is installed
+node -v
+
+# Check your NPM version
+npm -v
+\`\`\``,
+  },
+};
+
+export default {
+  component: CodeBox,
+  render: (_, { loaded: { Content } }) => Content,
+  loaders: [
+    async ({ args }) => {
```

Comment:
I would rather that we export the part of serialize of MDX of `next.dynamic.mjs` to a different function and then simply reimport it here.

Otherwise we can easily forget about this.

Reply:
I was able to do it but I exported it in a different file, because `next.dynamic.mjs` has some node-related imports.

Reply:
Gotcha